src/Uno.UWP/Media/Capture/CameraCaptureUI.Android.cs

```diff
@@ -0,0 +1,207 @@
+﻿#if __ANDROID__
```

Comment:
Not required?

Reply:
Uno.UI has no `.Android` implicit globbing for now.